Gemfile

```
@@ -8,11 +8,12 @@ gemspec
 
 gem 'benchmark-ips'
 gem 'bump'
-gem 'maxitest'
 gem 'memory_profiler'
 gem 'rake'
+
+gem 'maxitest'
+gem 'webmock', '~> 3.23'
```

should not be needed
```suggestion
gem 'webmock'
```

---

I have learned that  is always good to add a minimum version so you know you want downgrade by mistake. Also now you can run bundle update and things should work but with depricitions  if any. Then do the major bump up. 

but I will remove it if you want

---

bundler warns on downgrades so I usually don't add any
it adds a lot of clutter when every gem has a version :)
... just leave it I can remove them all whenever they cause an issue